.github/workflows/test.yml

```
@@ -31,7 +31,7 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - run: curl -LsSf https://astral.sh/uv/install.sh | sh
+    - uses: astral-sh/setup-uv@v3 with:   enable-cache: true
```

actionlint

could not parse as YAML: yaml: line 34: mapping values are not allowed in this context

(syntax-check)